File: Java/Introduction/NameClassSchool.java
//write a program to accept your name, class, division, school and print them
import java.util.Scanner;
public class NameClassSchool {
	public static void main(String[] args) {
		 Scanner myObj = new Scanner(System.in);
		 System.out.println("Enter your name");
		 String name = myObj.nextLine();
		 System.out.println("Enter your class and division");
		 String classdiv = myObj.nextLine();
		 System.out.println("Enter your school");
		 String school = myObj.nextLine();
		 System.out.println("Your name is " + name);
		 System.out.println("Your class and division is " + classdiv);
		 System.out.println("Your school is " + school);
	}
}	 

File: C++/Introduction/NameClassSchool.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
  string nameper;
  string classdiv;
  string school;
  cout << "Enter your name";
  getline (cin, nameper);
  cout << "Enter your class and division";
  getline (cin, classdiv);
  cout << "Enter your school";
  getline (cin, school);
  cout << "Your name is " << nameper << ". Your class and division is " << classdiv << ". Your school is " << school;
  return 0;
}
